src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -93,8 +95,17 @@ bool GetTrustedPlatformAssembliesList(const std::string& tpaDirectory, bool isNa
     return true;
 }
 
-// TODO: Figure out if runtimeDirectory should have a trailing "/".  Need to look at what happens on Windows.
-void* LoadCoreClr(std::string& runtimeDirectory)
+void* pLibCoreClr = nullptr;
```

Comment:
consider putting all but `CallApplicationMain` in the unnamed namespace to hide it from everyone else (won't be visible outside this translation unit) just for safety